config/environments/development.rb

```diff
@@ -64,4 +64,6 @@
 
   # Let Faker load its :en text
   config.i18n.enforce_available_locales = false
+
+  Rails.application.routes.default_url_options = { host: ENV['HOST_NAME'], port: ENV['PORT'] }
```

Comment:
Je crois que c’est la même chose que `config.routes.default_url_options` vu qu'on est dans un block `configure`? D’ailleurs, peut-être que cette option mériterait d'être un peu plus haut dans le fichier, c'est un peu important comme réglage. (idem dans production.rb)

Reply:
Ca ne fonctionne pas comme ça, à priori ça set la variable `config.action_controller.default_url_options` mais on ne peut pas l'écrire comme ça on est obligé de l'écrire comme je l'ai fait. Par contre on peut enlever `config.action_mailer.default_url_options` car elle set les deux. 
[https://stackoverflow.com/questions/36792278/where-to-set-host-for-rails-application-routes-url-helpers](https://stackoverflow.com/questions/36792278/where-to-set-host-for-rails-application-routes-url-helpers)